samples/README.md

```diff
@@ -0,0 +1,50 @@
+# Overview
+
+This sample application provides an example usage of the display ILI9163C from ILITEK.
+The example includes PWM control for backlight dimming and displays text and a counter on the screen.
+
+- Displays "6TRON BY CATIE!" on the screen. 
+- Shows an incrementing counter at the bottom of the screen. 
+- Uses PWM to control the backlight brightness in a pulsing pattern. 
+- Refreshes the display content periodically using LittlevGL (LVGL).
+
+> [!NOTE]
+>  Ensure the display and PWM hardware configurations match the setup below for correct operation.
+
+# Requirements
+
+- Hardware: Zest Display LCD shield with ILI9163C display and any compatible Zest Core board.
```

Comment:
This driver only depends on ILI9163C. It is not a depot where the shield should be mentionned.

Reply:
Resolved in 1ea59ed680dc881d3afea2fa9cafe1a5cf060ca0.